frontend/app/model/Eventbrite.scala

```diff
@@ -147,18 +147,34 @@ object Eventbrite {
 
   object RichEvent {
     implicit def eventToEBEvent(event: RichEvent) = event.event
+
     implicit def eventOptToEBEventOpt(eventOpt: Option[RichEvent]) = eventOpt.map(_.event)
   }
 
-  case class GuLiveEvent(event: EBEvent) extends RichEvent {
-    val imgUrl = Config.eventImageUrlPath(event.id) + "/{width}{pixel_ratio}.jpg"
+  case class EventImage(assets: List[Asset], metadata: Metadata)
+
+  case class GuLiveEvent(event: EBEvent, image: Option[EventImage]) extends RichEvent {
+
+    val imgUrl = image.flatMap(_.assets.headOption).fold(Config.gridConfig.fallbackImageUrl) { asset =>
+      val file = asset.secureFile.getOrElse(asset.file)
+      val regex = "\\d+.jpg".r
+      regex.replaceFirstIn(file, "{width}.jpg")
+    }
+
+    private val widths = image.fold(List.empty[Int])(_.assets.map(_.dimensions.width))
+
+    val availableWidths = widths.mkString(",")
+
+    val imageMetadata = image.map(_.metadata)
+
     val socialImgUrl = {
       // get the largest available image (for social media etc)
       // this mirrors the URL structure generated by Imager.js
       // so any changes there need to be reflected here
-      val largestWidth = Config.eventImageWidths.toArray.last
-      val largestRatio = Config.eventImageRatios.toArray.last
-      Config.eventImageUrlPath(event.id) + "/" + largestWidth + "-" + largestRatio + "x.jpg"
+      image.fold(Config.gridConfig.fallbackImageUrl) { img =>
+        val asset = img.assets.find(_.dimensions.width == widths.max)
+        asset.map(_.file).getOrElse(Config.gridConfig.fallbackImageUrl)
```

Comment:
I think we could remove the duplicate `Config.gridConfig.fallbackImageUrl` here
```
val socialImgUrl = image.flatMap(_.assets.find(_.dimensions.width == widths.max)).fold(Config.gridConfig.fallbackImageUrl)(_.file)
```

Or something like that